protocol.html

```diff
@@ -902,6 +902,8 @@ <h3 property="schema:name">Security Considerations</h3>
                   <p>Servers <a href="#cors-server">disable all cross-origin protections</a> in browsers because resource access is governed explicitly by the <a href="#authorization">Authorization</a> component. As such, servers cannot rely on browser-based cross-origin protection mechanisms for determining the authentication status or representation of a resource. In particular, servers are strongly encouraged to ignore HTTP cookies from untrusted origins. Additional security measures can be taken to prevent metadata in error responses from leaking. For instance, a malicious application could probe multiple servers to check whether the response status code is <code>401</code> or <code>403</code>, or could try to access an error page from an intranet server within the user agent’s private network to extract company names or other data. To mitigate this, when a request from an untrusted <code>Origin</code> arrives, the server may want to set the status code of error responses to <code>404</code> and/or anonymize or censor their contents.</p>
 
                   <p>Servers are encouraged to use TLS connections to protect the contents of requests and responses from eavesdropping and modification by third parties. Unsecured TCP connections without TLS may be used in testing environments or when the data pod is behind a reverse proxy that terminates a secure connection.</p>
+
+		  <p>Even though servers must support write operations, not all deployments are required to allow write operations for all resources. For security, servers should indicate any such restrictions on HTTP methods by using the <code>Allow</code> header and should respond to methods they do not allow using the <code>405</code> status code for the relevant resources.</p>
```

Comment:
```suggestion
```

I don't see this as a security consideration. It is rather about affordances.

I think 405 is already sufficiently covered at the beginning of #reading-writing-resources:

>Servers MUST respond with the 405 status code to requests using HTTP methods that are not supported by the target resource.

Reply:
Alright, it certainly works for me.